acceptance-tests/runner/scripts/start-selenium.sh

```diff
@@ -6,7 +6,7 @@ $SCRIPT_DIR/stop-selenium.sh
 echo ""
 echo " Starting Selenium Docker container..."
 echo ""
-docker run -d --name blueo-selenium -p 15900:5900 -p 7990:7990 -p 7999:7999 -p 4444:4444 -e no_proxy=localhost selenium/standalone-chrome-debug:3.6.0-bromine > /dev/null
+docker run -d --name blueo-selenium -p 15900:5900 -p 7990:7990 -p 7999:7999 -p 4444:4444 -e no_proxy=localhost selenium/standalone-chrome-debug:3.13.0-argon > /dev/null
```

Comment:
I remember back in the day, the first time I thought to name a bunch of products after the elements... I thought I was so clever and unique 😂